shell/common/gin_helper/locker.h

```diff
@@ -22,13 +22,7 @@ class Locker {
   Locker& operator=(const Locker&) = delete;
 
  private:
-  void* operator new(size_t size);
```

Comment:
@ckerr these were declared here as private to prevent the `Locker` class from being allocated on the heap

Reply:
https://github.com/electron/electron/pull/39847